homeassistant/components/bluesound/strings.json

```diff
@@ -1,4 +1,25 @@
 {
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        }
+      },
+      "confirm": {
+        "title": "Discover Bluesound player",
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      }
+    },
+    "abort": {
```

Comment:
I think zeroconf also requires the `flow_in_progress` translation key

Reply:
I cannot find `flow_in_progress`. Do you mean `already_in_progress`? That one seems to be thrown in `async_set_unique_id`.

Reply:
- Yes! I am on mobile so couldn't check the code